libraries/c_sdk/standard/mqtt/CMakeLists.txt

```diff
@@ -3,7 +3,13 @@ afr_module(INTERNAL)
 set(src_dir "${CMAKE_CURRENT_LIST_DIR}/src")
 set(inc_dir "${CMAKE_CURRENT_LIST_DIR}/include")
 set(test_dir "${CMAKE_CURRENT_LIST_DIR}/test")
-
+if( EXISTS ${AFR_MODULES_DIR}/coreMQTT/source )
+    set(transport_interface_dir "${AFR_MODULES_DIR}/coreMQTT/source/interface")
+elseif( EXISTS ${AFR_MODULES_DIR}/coreHTTP/source )
+    set(transport_interface_dir "${AFR_MODULES_DIR}/coreHTTP/source/interface")
```

Comment:
How come we need to check coreHTTP too? I think coreMQTT is already SSoT for transport interface plus this build is for MQTT.

Reply:
Thanks for pointing that out, fixed in 6e29b18539d2f1145874f40d9661a458090c93b1